af/insertstmt/prepared.go

```diff
@@ -0,0 +1,115 @@
+package insertstmt
+
+import (
+	"errors"
+	"github.com/taosdata/driver-go/v2/af/param"
+	taosError "github.com/taosdata/driver-go/v2/errors"
+	"github.com/taosdata/driver-go/v2/wrapper"
+	"sync"
+	"unsafe"
+)
+
+type PreparedStmt struct {
+	taos unsafe.Pointer
+	stmt unsafe.Pointer
+	sync.RWMutex
+	setTableName bool
+}
+
+func NewPreparedStmt(stmt, taos unsafe.Pointer) *PreparedStmt {
+	return &PreparedStmt{stmt: stmt, taos: taos}
+}
+
+func (p *PreparedStmt) SetTableName(name string) error {
+	if p.setTableName {
+		return errors.New("duplicate set table name")
+	}
+	p.Lock()
+	defer p.Unlock()
+	code := wrapper.TaosStmtSetTBName(p.stmt, name)
+	err := taosError.GetError(code)
+	if err != nil {
+		return err
+	}
+	p.setTableName = true
+	return nil
+}
+
+func (p *PreparedStmt) SetSubTableName(name string) error {
+	if p.setTableName {
+		return errors.New("duplicate set table name")
+	}
+	p.Lock()
+	defer p.Unlock()
+	code := wrapper.TaosLoadTableInfo(p.taos, []string{name})
```

Comment:
> `load_table_info` should be called explicitly with table names string `tb1,tb2,tb3...`, not for single sub table name. Show the usage in documents other than hide in methods.

I think in the interface design uploaded into a comma-delimited string instead of an array is not very elegant even say ugly!
especially in the method call rather than compromise for the convenience of configuration.

Reply:
> `load_table_info` should be called explicitly with table names string `tb1,tb2,tb3...`, not for single sub table name. Show the usage in documents other than hide in methods.

fixed.
i'm talking about the load_table_info interface design of C